recipes-dotnet/coreclr/coreclr/toolchain.patch

```diff
@@ -20,16 +20,16 @@ index 84ae2a5..17fdf15 100644
  
  ## Specify the toolchain
 -set(TOOLCHAIN "arm-linux-gnueabihf")
-+set(TOOLCHAIN "arm-target-linux-gnueabi")
++set(TOOLCHAIN "arm-poky-linux-gnueabi")
```

Comment:
Doh, I was also fooled by the generic name.
Ok, I'm not running original poky of course, I have customizations, but I didn't change to distribution name.

I guess the options are:
1) do_unpack_append () { sed -i s/arm-linux-gnueabihf/${YOCTO VARIABLES.../ }
2) template stuff on toolchain.patch.
Any preference?